### ISSUE TITLE
no flag after run the example flow

### ISSUE THREAD

**Issue by hanetyb** [SeanScripts/ComfyUI-PixtralLlamaMolmoVision#17]
could you please give some advice? whats noflag
![1734189595178](https://github.com/user-attachments/assets/b134b1f6-edfd-47f3-8666-2ef80499a3dc)

![image](https://github.com/user-attachments/assets/e0b90ea0-6c60-4303-9b62-c8505467d867)

**Comment by SeanScripts** [comment]
Apparently this was added in Python 3.11, so I'm guessing you're using
3.10. I might just update it to use 0 instead, should be equivalent to no
flag...

**Comment by hanetyb** [comment]
yes, it does. 3.10.14...the flag =0, ii put 0, but it seem same error. ..   i could try to update python 11. not sure any other incompliance happen.

btw. thanks for your  advice.
![image](https://github.com/user-attachments/assets/6c8b1c1e-9d00-4ffe-88b6-826c70008586)

**Comment by SeanScripts** [comment]
I've updated it to use 0 instead of `re.NOFLAG`, so hopefully that fixes it for Python 3.10
